MemoryMeter.c

```diff
@@ -39,7 +39,10 @@ static void MemoryMeter_updateValues(Meter* this) {
    this->values[MEMORY_METER_SHARED] = NAN;
    this->values[MEMORY_METER_COMPRESSED] = NAN;
    this->values[MEMORY_METER_AVAILABLE] = NAN;
-   Platform_setMemoryValues(this);
+
+   double usedNumber = 0;
```

Comment:
I don't think adding "Number" here adds any value - what was wrong with the existing "used" variable name?